sky/backends/cloud_vm_ray_backend.py

```diff
@@ -1991,9 +1991,21 @@ def provision_with_retries(
                     cloud_user = None
                 else:
                     cloud_user = to_provision.cloud.get_current_user_identity()
+
+                requested_features = self._requested_features.copy()
+                # Skip stop feature for Kubernetes jobs controller.
+                if isinstance(to_provision.cloud, clouds.Kubernetes
+                             ) and controller_utils.Controllers.from_name(
+                                 cluster_name
+                             ) == controller_utils.Controllers.JOBS_CONTROLLER:
```

Comment:
nit: would be nice to add `()`
```suggestion
                if (isinstance(to_provision.cloud, clouds.Kubernetes
                             ) and controller_utils.Controllers.from_name(
                                 cluster_name
                             ) == controller_utils.Controllers.JOBS_CONTROLLER):
```